src/test/ui/intrinsics/panic-uninitialized-zeroed.rs

```diff
@@ -65,59 +65,111 @@ fn main() {
             || mem::uninitialized::<!>(),
             "attempted to instantiate uninhabited type `!`"
         );
+        test_panic_msg(
+            || mem::uninitialized::<[!; 2]>(),
+            "attempted to instantiate uninhabited type `[!; 2]`"
+        );
```

Comment:
I always kept the tests for uninit and zeroed the same type directly next to each other; would be good if you could keep that structure.

Reply:
Sounds good. Fixed